bundles/org.openhab.binding.carnet/src/main/java/org/openhab/carnet/internal/services/CarNetVehicleServiceStatus.java

```diff
@@ -0,0 +1,264 @@
+            // MAINT_ALARM_INSPECTION+MAINT_ALARM_OIL_CHANGE + MAINT_ALARM_OIL_MINIMUM
+            logger.debug("{}: Maintenance required: {} has incorrect pressure", thingId, definition.symbolicName);
+            return true;
+        }
+        if (definition.symbolicName.contains("AD_BLUE_RANGE") && (Integer.parseInt(field.value) < 1000)) {
+            logger.debug("{}: Maintenance required: Ad Blue at {} (< 1.000km)", thingId, field.value);
+            return true;
+        }
+        return false;
+    }
+
+    private boolean checkLocked(CNStatusField field, ChannelIdMapEntry definition) {
+        if (definition.symbolicName.contains("LOCK")) {
+            boolean result = (definition.symbolicName.contains("LOCK2") && field.value.equals(String.valueOf(2)))
+                    || (definition.symbolicName.contains("LOCK3") && field.value.equals(String.valueOf(3)));
+            if (!result) {
+                logger.debug("{}: Vehicle is not completetly locked: {}", thingId, definition.channelName);
+                return false;
+            }
+        }
+        return true;
+    }
+
+    private boolean checkWindows(CNStatusField field, ChannelIdMapEntry definition) {
+        if (definition.symbolicName.contains("WINDOWS") && definition.symbolicName.contains("STATE")
+                && !field.value.equals(String.valueOf(3))) {
+            logger.debug("{}: Window {} is not closed", thingId, definition.channelName);
+        }
+        return true;
+    }
+
+    private boolean checkTires(CNStatusField field, ChannelIdMapEntry definition) {
+        if (definition.symbolicName.contains("TIREPRESS") && definition.symbolicName.contains("CURRENT")
+                && !field.value.equals(String.valueOf(1))) {
+            logger.debug("{}: Tire pressure for {} is not ok", thingId, definition.channelName);
+        }
+        return true;
+    }
+
+    private void updateNumberChannel(Channel channel, ChannelIdMapEntry definition, CNStatusField field) {
+        State state = UnDefType.UNDEF;
+        String val = getString(field.value);
+        if (!val.isEmpty()) {
+            double value = Double.parseDouble(val);
+            if (value < 0) {
+                value = value * -1.0; // no egative values
+            }
+            BigDecimal bd = new BigDecimal(value);
+            if (definition.unit != null) {
+                ChannelIdMapEntry fromDef = definition;
+                fromDef = idMapper.updateDefinition(field, definition);
+                if ((fromDef.fromUnit != null) && !fromDef.fromUnit.equals(definition.unit)) {
+                    try {
+                        // Convert between units
+                        bd = new BigDecimal(fromDef.fromUnit.getConverterToAny(definition.unit).convert(value));
+                    } catch (UnconvertibleException | IncommensurableException e) {
+                        logger.debug("{}: Unable to covert value", thingId);
```

Comment:
You might want to log the exception's message.

Reply:
done